.travis.yml

```diff
@@ -102,21 +96,19 @@ before_deploy:
 
 deploy:
  - provider: releases
```

Comment:
Why change the key name?

Reply:
The Api_key and local-dir changes were because of info popups on travis' side about them defaulting to aliases for other variables

`
jobs.include.language: value sh is an alias for shell, using shell
deploy: key local-dir is not underscored, using local_dir
deploy: key api_key is an alias for token, using token
root: key matrix is an alias for jobs, using jobs 
`